src/main/kotlin/graphics/scenery/volumes/VolumeManager.kt

```diff
@@ -306,19 +305,68 @@ class VolumeManager(
         )
         segments[SegmentType.Accumulator] = SegmentTemplate(
             "AccumulateSimpleVolume.frag",
-            "vis", "sampleVolume", "convert", "sceneGraphVisibility"
+            "vis", "localNear", "localFar", "sampleVolume", "convert", "sceneGraphVisibility"
         )
 
         customSegments?.forEach { (type, segment) -> segments[type] = segment }
 
+        var triggered = false
         val additionalBindings = customBindings
-            ?: BiConsumer { _: Map<SegmentType, SegmentTemplate>, instances: Map<SegmentType, Segment> ->
-                logger.debug("Connecting additional bindings")
+            ?: MultiVolumeShaderMip.SegmentConsumer { _: Map<SegmentType, SegmentTemplate>,
+                                                  segmentInstances: Map<SegmentType, Segment>,
+                                                  volumeIndex: Int ->
+                logger.info("Connecting additional bindings")
+
+                if(!triggered) {
+                    segmentInstances[SegmentType.FragmentShader]?.repeat("localNear", n)
+                    segmentInstances[SegmentType.FragmentShader]?.repeat("localFar", n)
+                    triggered = true
+                }
+
+                logger.info("Connecting localNear/localFar for $volumeIndex")
+//                instances[SegmentType.FragmentShader]?.bind(
+//                    "localNear",
+//                    i,
+//                    instances[SegmentType.AccumulatorMultiresolution]
+//                )
+
+                segmentInstances[SegmentType.FragmentShader]?.bind(
+                    "localNear",
+                    volumeIndex,
+                    segmentInstances[SegmentType.Accumulator]
+                )
+
+//                instances[SegmentType.FragmentShader]?.bind(
+//                    "localFar",
+//                    i,
+//                    instances[SegmentType.AccumulatorMultiresolution]
```

Comment:
Need some help on that since you are more familiar with the bvv API. We did a version of this on the `vdi_integration` branch earlier https://github.com/scenerygraphics/scenery/blob/vdi_integration/src/main/kotlin/graphics/scenery/volumes/VolumeManager.kt#L338C1-L360C18. Does this look reasonable?